arviz/stats/stats.py

```diff
@@ -1209,6 +1213,14 @@ def summary(
            ...:     extend=False
            ...: )
 
+    Use ``stat_focus`` to change the focus of summary statistics obatined to median:
+
+    .. ipython::
+
+        In [1]: import arviz as az
+           ...: data = az.load_arviz_data("centered_eight")
+           ...: az.summary(data, stat_focus="median")
```

Comment:
```suggestion
        In [1]: az.summary(data, stat_focus="median")
```

the lines of code executed previously _in the same docstring_ are remembered, it is not necessary to import arviz again or to create `data`.